srcpkgs/igt-gpu-tools/template

```diff
@@ -1,20 +1,30 @@
 # Template file for 'igt-gpu-tools'
 pkgname=igt-gpu-tools
-version=1.23
-revision=2
+version=1.24
+revision=1
 build_style=meson
-hostmakedepends="flex gtk-doc pkg-config swig"
-makedepends="cairo-devel libXrandr-devel libXv-devel libkmod-devel
- libunwind-devel procps-ng-devel python3-devel libressl-devel"
+hostmakedepends="pkg-config flex peg python3-docutils gtk-doc"
+makedepends="libdrm-devel libkmod-devel procps-ng-devel libunwind-devel
+ elfutils-devel pixman-devel valgrind-devel cairo-devel xmlrpc-c-devel gsl-devel
+ alsa-lib-devel json-c-devel libXrandr-devel"
 short_desc="Tools for development and testing of the Intel DRM driver"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="MIT"
 homepage="https://gitlab.freedesktop.org/drm/igt-gpu-tools"
 distfiles="${XORG_SITE}/app/${pkgname}-${version}.tar.xz"
-checksum=4d4b086c513bace5c23d0889de3f42ac3ebd3d968c64dedae6e28e006a499ad0
+checksum=57357c72feeafc923c9cfd2d1234bd80e120fc7cc6099eac81158ec351a821bf
 
 lib32disabled=yes
-archs="i686 x86_64"
+archs="i686* x86_64*"
+
+case "$XBPS_TARGET_MACHINE" in
+	*-musl) configure_args+=" -Dbuild_tests=disabled -Dbuild_runner=disabled -Dbuild_docs=disabled"
+esac
+
+# Some default flag makes the build hang in a random point
+do_configure() {
+	meson build
```

Comment:
NO. figure out what causes the issue and revert it it configure_args. Currently you are just making sure that the build does not fail, but everything get's installed in wrong locations, you don't pass the musl configure_args, etc...

Reply:
Add `-Db_lto=false` to configure args and add a comment that it takes unusually long on glibc with lto on.

Reply:
Okay, you found it already out :D

Reply:
I was just doing that